firebase-auth/src/androidMain/kotlin/dev/gitlive/firebase/auth/credentials.kt

```diff
@@ -123,6 +124,7 @@ actual interface PhoneVerificationProvider {
     val activity: Activity
     val timeout: Long
     val unit: TimeUnit
+    fun onRecievedVerificationID(verificationID: String)
     fun codeSent(triggerResend: (Unit) -> Unit)
```

Comment:
why separate method, instead of the `codeSent`'s parameter?

Reply:
I don't want to change a method signature.  Also, less convoluted code is easier to read and maintain. Separation of concern. Separation of responsibility.

Reply:
Why does it matter? It doesn't seem like anyone is using `codeSent`.... because as soon as you try to use it - you run into this same problem (https://github.com/GitLiveApp/firebase-kotlin-sdk/issues/104)

Also, why is there `Unit` parameter in `triggerResend: (Unit) -> Unit) ` -- should be just `triggerResend: () -> Unit)` - a normal signature for a method that doesn't take any parameters 🤷

Reply:
I am not sure about the last part but I have worked with many sdks and the smaller the pieces the better.  Less siginture changes the better. 

Either way this update is needed and had been needed for a very long time.  If it doesn't happen soon I will have to shift into building and maintaining my own SDK for this which is what I am trying to avoid.

Reply:
api should follow android sdk apart from the kotlin-first design principles defined in the readme, is this the case for this?
If you paste in links to the relevant firebase sdk documentation  I would be happy to confirm it for you

Reply:
override fun onCodeSent(
                verificationId: String,
                token: PhoneAuthProvider.ForceResendingToken
            ) {

https://github.com/firebase/snippets-android/blob/076236ee54e3b6c1bab0152c52fdb3ebc5c0640a/auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt#L39-L81

Line 68

Reply:
Android applications need that ID or they cant complete the phone code auth.

Reply:
Someone else may need this so I'll leave it open but I am pivoting away from this.